tests/scripts/task_demo_microtvm.sh

```diff
@@ -34,6 +40,18 @@ FVP_PATH="/opt/arm/FVP_Corstone_SSE-300_Ethos-U55"
 CMAKE_PATH="/opt/arm/cmake/bin/cmake"
 FREERTOS_PATH="/opt/freertos/FreeRTOSv202112.00"
 
- timeout 5m ./run_demo.sh --fvp_path $FVP_PATH --cmake_path $CMAKE_PATH
- timeout 5m ./run_demo.sh --fvp_path $FVP_PATH --cmake_path $CMAKE_PATH --freertos_path $FREERTOS_PATH
+timeout 5m ./run_demo.sh --fvp_path $FVP_PATH --cmake_path $CMAKE_PATH > out.log
```

Comment:
Should redirection be done to a tmp file? Also, a clean up of the log should be useful.
Example: https://github.com/apache/tvm/blob/476d9d1d41980ad35cd6a4d044c17fea2210f352/tests/scripts/task_python_docs.sh#L53

Reply:
Thanks, agreed that makes more sense